output/cloud/output_test.go

```diff
@@ -396,28 +361,6 @@ func testCloudOutputStopSendingMetric(t *testing.T, stopOnError bool) {
 	builtinMetrics := metrics.RegisterBuiltinMetrics(metrics.NewRegistry())
 
 	tb := httpmultibin.NewHTTPMultiBin(t)
-	tb.Mux.HandleFunc("/v1/tests", http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
-		body, err := io.ReadAll(req.Body)
-		require.NoError(t, err)
-		data := &cloudapi.TestRun{}
-		err = json.Unmarshal(body, &data)
-		require.NoError(t, err)
-		assert.Equal(t, "my-custom-name", data.Name)
```

Comment:
I don't think this test name logic or the validity of the JSON is checked anywhere now, you should probably add it to `TestOutputStartStopTestCreationWithConfigOverwrite`.

Reply:
The integration is aready covered by e2e tests, instead the unit I think it is more appropriate to be done in the `cloudapi` pkg. Done in https://github.com/grafana/k6/pull/3061/commits/97f22c76cc81d61dc9db0a0d5bd27fa62070fbb6

Reply:
Hmm I don't see where in the e2e tests we are checking if setting `options.ext.loadimpact.name` to something like `my-custom-name` will result in the backend getting that info in the `POST /v1/tests` API request :confused: I see these 2 places and the default handler:
https://github.com/grafana/k6/blob/9e3359b7dd4b6dec7440d9e2cac6948b3ed64e4a/cmd/tests/cmd_run_test.go#L1687-L1702C6
https://github.com/grafana/k6/blob/9e3359b7dd4b6dec7440d9e2cac6948b3ed64e4a/cmd/tests/cmd_run_test.go#L1662-L1667
https://github.com/grafana/k6/blob/9e3359b7dd4b6dec7440d9e2cac6948b3ed64e4a/cmd/tests/cmd_cloud_test.go#L18-L24

But we are not even checking if we are getting valid JSON there :disappointed: If you add that check to an e2e test or two (or point me to an existing one I missed), then :+1: :+1: for getting it out of here. It will be much better suited to an e2e test indeed.

Reply:
>Hmm I don't see where in the e2e tests we are checking if setting options.ext.loadimpact.name

Sorry, I misunderstood this part from the previous comment. Should be done now.

Edited the current e2e for adding the coverage https://github.com/grafana/k6/pull/3061/commits/2cffc10c88ef093166c0b9978d437b7257a7c1de